airbyte-cdk/python/airbyte_cdk/models/airbyte_protocol.py

```diff
@@ -1,7 +1,3 @@
-#
```

Comment:
the changes in this file look sus. formatting / linting didn't run?

Reply:
Yes, I manually run the generate script and it created this changes. I have posted on the connector channel about it.

Reply:
they responded. sounds like you just need to run build.

Reply:
yeah, I managed to finished on build. it is fixed